pkg/mutate/server.go

```diff
@@ -18,13 +27,57 @@ type MutationServer struct {
 	mux    *http.ServeMux
 	server *http.Server
 
-	config *MutationConfig
+	webhookClient client.Client
+	config        *MutationConfig
+}
+
+func getWebhookClient(ctx context.Context) (client.Client, error) {
```

Comment:
Can be this method factorized in the utils package externalizing the scheme? I think it can be reused outside the webhook.

Reply:
Sure, I have created a new method in liqo utils.